wiki/Replay/en.md

```diff
@@ -0,0 +1,285 @@
+<!-- wiki -->
+<!-- [Auto wikilink]: /wiki/Game_Modifiers#-auto "Auto game modifier" -->
+[Auto wikilink]: /wiki/Game_Modifiers "more info can be found on Game Modifiers under Auto"
+[Cinema wikilink]: /wiki/Game_Modifiers "more info can be found on Game Modifiers under Cinema"
+[No Fail wikilink]: /wiki/Game_Modifiers "more info can be found on Game Modifiers under No Fail"
+[Relax wikilink]: /wiki/Game_Modifiers "more info can be found on Game Modifiers under Relax"
+[Auto Pilot wikilink]: /wiki/Game_Modifiers "more info can be found on Game Modifiers under Auto Pilot"
+[Perfect wikilink]: /wiki/Game_Modifiers "more info can be found on Game Modifiers under Perfect"
+[osr wikilink]: /wiki/osu!_File_Formats/Osr_(file_format)/ ".osr file format"
+[Chat Console wikilink]: /wiki/Chat_Console/ "Chat Console"
+[Extended Chat Console wikilink]: /wiki/Chat_Console/ "more info can be found on Chat Console under Extended Chat Console"
+
+<!-- image -->
+[Replay Auto]: ./Replay_Auto.jpg "Replay with Auto, offline and online account play respectively"
+[Replay discussion]: ./Replay_discussion.jpg "Discussion button options during a replay"
+[Replay discussion comment]: ./Replay_discussion_comment.jpg "Making a comment"
+[Replay discussion example]: ./Replay_discussion_example.jpg "An example of comment flow"
+[Replay Solo]: ./Replay_Solo.jpg "A local replay"
+[Replay failed]: ./Replay_failed.jpg "Left is a regular game over screen and right is a failed replay game over screen"
+[Replay Multi]: ./Replay_Multi.jpg "Left is the original Multi play and right is an exported Multi replay"
+[Replay Multi fail]: ./Replay_Multi_fail.jpg "A failed Multi replay will still be played."
+[Replay Auto on]: ./Replay_Auto_on.jpg "Replay with Auto when connected to Bancho"
+[Replay server]: ./Replay_server.jpg "Left is downloading the server replay and right is the replay in question."
+[Replay preview result]: ./Replay_preview_result.jpg "An example of preview gameplay's result screen"
+
+# Replay
+
+## Introduction
+
+Replay is a direct emulation of a play, either by a player or a perfect play like [Auto][Auto wikilink] mod.
+As long as the local replay emulation file with a file extension, `.osr`, was available with the difficulty/beatmap it reference or a top _Global leaderboard_ play was downloaded, a complete replay can be constructed by osu! to be watched.
+
+## Replay options
+
+![Replay with Auto, offline and online account play respectively][Replay Auto]
+
+_Left is offline replay; Right is online replay_
+
+To rewatch a replay, hold the custom default grave/tilde (`` `/~ ``) key (can be changed in `Options` sidebar; inside _Change keyboard bindings_ as **Quick Retry (hold briefly)**) or `Ctrl`+`R` for a while.
+The screen should darken and a sound will be played when the quick restart was successful.
+Releasing the key early will fail the restart.
```

Comment:
~~Auto will suddenly miss all the hitcircles for no reason and get 50 on sliders just because I open foobar2000 together with osu! and NO SOUNDS ON osu!~~

You go back to the replay again.

Redundant note.